cloud/stack/filterstatus.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+package stack
+
+import (
+	"github.com/terramate-io/terramate/errors"
+	"golang.org/x/exp/slices"
+)
+
+const (
+	// ErrInvalidFilterStatus represents an invalid filter stack status error.
+	ErrInvalidFilterStatus errors.Kind = "invalid filter stack status"
+)
+
+type (
+	// FilterStatus represents the query paramater used for filtering stacks via the API
```

Comment:
```suggestion
	// FilterStatus represents the query parameter used for filtering stacks via the API
```

Reply:
Fixed in https://github.com/terramate-io/terramate/pull/1223/commits/074f235065a2efea2d5f284c89912e7e5076b5ad will rebase when review is complete